src/cascadia/TerminalSettingsEditor/EditAction.xaml

```diff
@@ -0,0 +1,703 @@
+    </Page.Resources>
+
+    <Border MaxWidth="{StaticResource StandardControlMaxWidth}"
+            Margin="{StaticResource SettingStackMargin}">
+        <Grid MaxWidth="600"
+              Margin="{StaticResource SettingStackMargin}"
+              HorizontalAlignment="Left"
+              ColumnSpacing="16"
+              RowSpacing="8">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
+            <TextBlock x:Uid="Actions_Name"
+                       Grid.Row="0"
+                       Grid.Column="0"
+                       VerticalAlignment="Center" />
+            <TextBox Grid.Row="0"
+                     Grid.Column="1"
+                     Width="300"
+                     HorizontalAlignment="Left"
+                     PlaceholderText="{x:Bind ViewModel.DisplayName, Mode=OneWay}"
+                     Text="{x:Bind ViewModel.Name, Mode=TwoWay}" />
+            <TextBlock x:Uid="Actions_ShortcutAction"
+                       Grid.Row="1"
+                       Grid.Column="0"
+                       VerticalAlignment="Center" />
+            <ComboBox Grid.Row="1"
+                      Grid.Column="1"
+                      VerticalAlignment="Center"
+                      ItemsSource="{x:Bind ViewModel.AvailableShortcutActions, Mode=OneWay}"
+                      SelectedItem="{x:Bind ViewModel.ProposedShortcutAction, Mode=TwoWay}" />
+            <TextBlock x:Uid="Actions_Arguments"
+                       Grid.Row="2"
+                       Grid.Column="0"
+                       VerticalAlignment="Center"
+                       Visibility="{x:Bind ViewModel.ActionArgsVM.HasArgs, Mode=OneWay}" />
+            <ListView Grid.Row="2"
+                      Grid.Column="1"
+                      Margin="-28,0,0,0"
+                      AllowDrop="False"
+                      CanDragItems="False"
+                      CanReorderItems="False"
+                      ChoosingItemContainer="_choosingItemContainer"
+                      ItemsSource="{x:Bind ViewModel.ActionArgsVM.ArgValues, Mode=OneWay}"
+                      SelectionMode="None" />
+            <TextBlock x:Uid="Actions_Keybindings"
+                       Grid.Row="3"
+                       Grid.Column="0"
+                       VerticalAlignment="Center" />
+            <ListView Grid.Row="3"
+                      Grid.Column="1"
+                      ItemTemplate="{StaticResource KeyChordTemplate}"
+                      ItemsSource="{x:Bind ViewModel.KeyChordViewModelList, Mode=OneWay}"

```

Comment:
😈 Evil question.

Consider that I have "Clear buffer" already bound to "ctrl+shift+k". 

Now, if I go to the Actions page and "add new" action. Then set action type to "Clear buffer". Shouldn't the existing key bindings be listed?

![image](https://github.com/user-attachments/assets/49100968-0a19-4d27-afd0-da2c00d26cef)

Reply:
Yeah this is weird - you have created a new command object with a new ID so not showing the existing key bindings is _technically_ correct, however by _feel_ since your new command is doing exactly the same thing as an existing command it would make sense to show the existing keybindings. So we would probably need to implement some logic every time the args/action type changes to check if it matches an existing command already and then update this page, but I think that makes more sense as a followup and for now if a user is trying to add keybindings to an existing command they should just... navigate to that one instead of creating a new command that does exactly the same thing.